src/DataViews/LiveFunctionsDataView.cpp

```diff
@@ -464,14 +479,12 @@ void LiveFunctionsDataView::AddFunction(uint64_t function_id,
                                         orbit_client_protos::FunctionInfo function_info) {
   functions_.insert_or_assign(function_id, std::move(function_info));
   indices_.push_back(function_id);
-  timer_durations_.insert_or_assign(function_id, GetFunctionTimerDurations(function_id));
```

Comment:
Shouldn't `UpdateTimerDurations();` be called here to have the same effect?

I'm a bit lost with the change to `GetFunctionTimerDurations` as I don't really know the code, we definitely need more pairs of eyes here.

Reply:
`AddFunction` is called on `OnDataChanged` for each function. Previously, I've been calling `GetFunctionTimerDurations` to collect all the timers related to the function. Now I have `UpdateTimerDurations` which does the same thing for for all the functions.

Reply:
Oh, and I call `GetFunctionTimerDurations` in `OnDataChanged`.